mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1333,6 +1332,8 @@ def predict_with_latency(X_copy):
 
         X_copy = self.X.copy_to_avoid_mutation()
         if self.model is not None:
+            _logger.info("Computing model predictions.")
+
```

Comment:
is this needed? we already have in https://github.com/mlflow/mlflow/pull/10107

Reply:
Yes moving it down since we only should log this if we are doing predictions(only if we have a model). Does that makes sense?

Reply:
Spoke offline. This change looks good :)